_data/faq.yml

```diff
@@ -115,10 +115,126 @@ it:
+      the acquisition and reuse of software, be it software
+      for teaching or software for administrative purposes?
+    answer: |
+      Recipients of the provisions of Articles 68 and 69 of the CAD
+      are the public administrations referred to in art. 1, paragraph 2, of the
+      legislative decree March 30, 2001, n. 165, in respect of the division of
+      competence of art. 117 of the Constitution; among these, there are the
+      institutes and schools of all levels and educational institutions.
+      The combined provisions of Articles 68 and 69 of the CAD do not operate any
+      distinction about the type of purpose of a software. The object of the
+      obligation established by the above mentioned provisions is the software of
+      ownership of public administration.
+  item2:
+    question: In which cases is the PA considered owner of a software?
+    answer: |
+      The administration must be considered owner of a software realized on
+      its own specific indications every time:  
+      <ul>
+      <li>it commissioned the solution, through a procurement contract
+      or other negotiating circumstances, and the contract foresees the acquisition
+      of all intellectual and industrial property rights
+      on the software developed on its client's behalf or in any case it does not
+      attribute such ownership by the contractor or third parties;</li>.
+      <li>the software has been realized by internal resources of the administration.</li>
+      </ul>
+  item3:
+    question: Do articles 68 and 69 of CAD apply before any software procurement?
+    answer: |
+      Articles 68 and 69 of the CAD apply in any case of solutions acquisition by
+      a Public Administration, software or parts of it
+      regardless of the type of procedure.
+      For more information about CAD see <a
+      href="https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2018-09-28/_rst/capo6_art68.html">this
+      link</a>.
+  item4:
+    question: Can non-open source software owned by a PA be put to Reuse?
+    answer: |
+      It is necessary to distinguish between software requested and specifically
+      developed for a Public Administration (or developed within it)
+      and software of commercial nature of which an administration has instead
+      acquired a right of use license. Only the software belonging to the first
+      category is subject to art. 69 of the CAD on reuse: for such solutions
+      administrations must obligatorily acquire full ownership of the source code
+      and must proceed with its release in open source. Instead the commercial
+      software acquired by means of a use right license can not be put to reuse.
+  item5:
+    question: Which are the indications provided for the purposes of compliance
+      of art. 69 regarding the choice of the license?
+    answer: |
+      The "Guidelines on the acquisition and reuse of software for the public
```

Comment:
This is the correct English title: 
Guidelines on the acquisition and reuse of software for public administrations